mutiny-core/src/lib.rs

```diff
@@ -108,10 +118,17 @@ impl<S: MutinyStorage> MutinyWallet<S> {
 
         let node_manager = Arc::new(NodeManager::new(config.clone(), storage.clone()).await?);
 
+        // create nostr manager
+        let seed = node_manager.show_seed().to_seed("");
+        let xprivkey = ExtendedPrivKey::new_master(node_manager.get_network(), &seed)?;
+        let relays = vec!["wss://relay.damus.io".to_string()]; // todo make configurable
```

Comment:
Should we at least do a second just in case damus goes down?

Reply:
For nostr wallet connect you can only have one so just hard coded this one for now

Reply:
Oh that's dumb that you can only have one